### ISSUE TITLE
Google drive index script is not working

### ISSUE THREAD

**Issue by rsoorajs** [gja/cloudflare-worker-local#54]
[https://github.com/LeeluPradhan/G-Index/blob/master/Search/index.js](https://github.com/LeeluPradhan/G-Index/blob/master/Search/index.js) 

This script is not working.This is a `gdrive index` script.The index page is loading,but the files didnt loading..the page kept showing loading. can you please fix this ?

**Comment by bsggr78** [comment]
If we start the download and cancel it, the use of node ram reaches 90% and the system crashes. This bug must be resolved immediately.

@gja

**Comment by gja** [comment]
Sorry for the late response. But is this bug for the right project?

**Comment by bsggr78** [comment]
> Sorry for the late response. But is this bug for the right project?

yes right project because the code works correctly in cloudflare. but it causes ram leak in the local worker.

There is no problem until the download button is canceled, but if the download is started and canceled, nodejs uses the entire ram and the system becomes unusable.

**Comment by bsggr78** [comment]
https://github.com/alx-xlx/goindex/

I guess you can detect where the error is and fix it with this script.
